lib/ardb/cli/commands.rb

```diff
@@ -197,6 +205,8 @@ def run(argv, *args)
         migration = Ardb::Migration.new(Ardb.config, @clirb.args.first)
         migration.save!
         @stdout.puts "generated #{migration.file_path}"
+      rescue Ardb::InvalidDBFileError
+        raise
```

Comment:
@kellyredding - I agree with you. For some reason I had blinders on and didn't even consider this an option. Thanks.